### PR TITLE
[iOS TextInput] Disables system keyboard for TextInputType.none

### PR DESCRIPTION
**Pull request by LongCatIsLooong**
Continuation of https://github.com/flutter/engine/pull/26773
/cc @jpnurmi 

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I signed the [CLA].
- [ ] All existing and new tests are passing.
- [ ] The reviewer has submitted any presubmit flakes in this PR using the [engine presubmit flakes form] before re-triggering the failure.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[engine presubmit flakes form]: https://forms.gle/Wc1VyFRYJjQTH6w5A

### CODE REVIEW

File: shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm [hunk 1]
```diff
@@ -516,7 +524,12 @@ @implementation FlutterTextInputView {
   int _textInputClient;
   const char* _selectionAffinity;
   FlutterTextRange* _selectedTextRange;
+  UIInputViewController* _inputViewcController;
```
Comment (jpnurmi) [inline shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm:527]: a small typo

**Comment by jpnurmi** [comment]
Thank you!

**Review by justinmc — APPROVED** [review]
LGTM 👍 

Does Android already disable the system keyboard?

**Comment by LongCatIsLooong** [comment]
@justinmc the Android PR is here: https://github.com/flutter/engine/pull/26585

**Comment by chunhtai** [comment]
I assume developer uses this for custom keyboard? can you double check whether using the custom keyboard with the voice over on still pronounce feedback when you type?

**Comment by LongCatIsLooong** [comment]
@chunhtai I couldn't test voiceover on simulator (but the accessibility inspector shows the text field as a button, nothing seems to change in the inspector when I disable/enable the system keyboard). On device I don't have ways to enter new text but it pronounces the current content.

**Comment by chunhtai** [comment]
@LongCatIsLooong, the simulator is fine. For device, can you try to simulate a custom keyboard? for example, have a button that insert a character into the textfield when pressed, the voiceover should announce the character that got inserted

**Comment by LongCatIsLooong** [comment]
Wouldn't the button steal the focus from the text field in that case?

**Comment by chunhtai** [comment]
@LongCatIsLooong yes, It is the same for for system keyboard, too. TextField announce the key feedback even when the keyboard strokes have the focus.

**Comment by LongCatIsLooong** [comment]
Ok yeah It does pronounce the new character if I focus the text field first.

**Review by chunhtai — APPROVED** [review]
LGTM

**Pull request merged**